stock_picking_invoice_link/wizards/account_move_reversal.py

```diff
@@ -10,7 +10,7 @@ def reverse_moves(self, is_modify=False):
         """Link return moves to the lines of refund invoice"""
         action = super(
             AccountMoveReversal, self.with_context(force_copy_stock_moves=True)
-        ).reverse_moves()
+        ).reverse_moves(is_modify)
```

Comment:
Don't lose keyword arguments

```suggestion
        ).reverse_moves(is_modify=is_modify)
```

Reply:
Done @pedrobaeza